lib/configurators/aws/awsconfigurator.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/lib/config"
+	"github.com/gravitational/teleport/lib/configurators"
+)
+
+// BuildConfigurators reads the configuration and returns a list of
+// configurators. Configurators that are "empty" are not returned.
+func BuildConfigurators(flags configurators.BootstrapFlags) ([]configurators.Configurator, error) {
```

Comment:
Can we consider moving this function back to the `configurators` package?

This function was intended to provide multiple configurators to the caller. For example, if we add an Azure Configurator, it should return it if it isn't empty. Moving to the `aws` package makes it lose its purpose, only working as a "utility" function to get `config.FileConfig` before initializing the AWS configurator.